js/src/reports/metric-number.js

```diff
@@ -48,19 +53,52 @@ const MetricNumber = ( {
 		const formatFn = isCurrency ? formatAmount : formatNumber;
 
 		return {
-			value: formatFn( value ),
+			value: value === undefined ? unavalable : formatFn( value ),
 			prevValue: formatFn( prevValue ),
 		};
 	}, [ isCurrency, value, prevValue, formatNumber, formatAmount ] );
 
 	let markedLabel = label;
+	let infoText;
 
 	// Until ~Q4 2021, metrics for all programs, may lack data for free listings.
-	if ( missingFreeListingsData ) {
-		const infoText = __(
-			'This data is available for paid campaigns only.',
+	// And Free Listings API may not respond with data.
+	if ( missingFreeListingsData === MISSING_FREE_LISTINGS_DATA.FOR_METRIC ) {
+		infoText = __(
+			'This data is currently available for paid campaigns only.',
 			'google-listings-and-ads'
 		);
+	} else if (
+		missingFreeListingsData === MISSING_FREE_LISTINGS_DATA.FOR_REQUEST
+	) {
+		infoText = (
+			<>
+				{ createInterpolateElement(
+					__(
+						'This data is currently available for paid campaigns only.<br/><br/>Please try again later, or go to <link>merchants.google.com</link> to track your performance for Google Free Listings. ',
```

Comment:
I wanted to give the translators the opportunity to divide the text into as many (or none) lines as they find it applicable for their copy. Is that an anti-pattern? Or maybe you meant something like below?
```js
__(
'<p>This data is currently available for paid campaigns only.<p><p>Please try again later, or go to <link>merchants.google.com</link> to track your performance for Google Free Listings.</p>',
'google-listings-and-ads'
),
```

Reply:
I don't think the translators would care about those elements like `<p>` or `<br/>` or the UI; those elements might even be unnecessary noise to the translators. 

What I meant is something like this: 

```js
infoText = (
			<>
				<div>
					{ /* first sentence. */ }
					{ __(
						'This data is currently available for paid campaigns only.',
						'google-listings-and-ads'
					) }
				</div>
				<div>
					{ /* second sentence. */ }
					{ createInterpolateElement(
						__(
							'Please try again later, or go to <link>merchants.google.com</link> to track your performance for Google Free Listings.',
							'google-listings-and-ads'
						),
						{
							br: <br />,
							link: (
								<TrackableLink
									eventName="gla_google_mc_link_click"
									eventProps={ {
										context: 'reports',
										href: googleMCURL,
									} }
									type="external"
									target="_blank"
									href={ googleMCURL }
								/>
							),
						}
					) }
				</div>
			</>
		);
```

Note that I wrap the sentences with `div`s; it can be `<p>` or `<span>` depending on our UI needs, and it would be independent from the translators.

Reply:
> What I meant is something like this:

My concern here was that we are hard-coding the fact there are two sentences (no less, no more). Which in my opinion was more of a copywriter/translator decision on how much text would end up there and how it would be divided into sentences. For example, somebody would like to phrase it more like:
> "Please try directly at merchants.google.com, as this data is currently not available for free listings"

And they would not be able to do that, without leaving the awkward empty space before or after.

But maybe, given my make-public-service-components background, I'm just mixing the roles of translator and copywriter here, as we already have specified copy for two separate messages.

Reply:
Also, in my mental model, it was a single message: the explanation why the data may not be accurate.

Reply:
Does [`979e77b` (#749)](https://github.com/woocommerce/google-listings-and-ads/pull/749/commits/979e77b590c59a8f81e0f799e34c2fe8fc4e9486) look better for you?

Reply:
Yeah I think it does. 🙂